test/mix/tasks/phoenix.gen.channel_test.exs

```diff
@@ -14,7 +14,7 @@ defmodule Mix.Tasks.Phoenix.Gen.ChannelTest do
   end
 
   test "generates channel" do
-    in_tmp "generates channel", fn ->
+    in_tmp "generates deprecated channel", fn ->
```

Comment:
No worries. `in_tmp` has been pesky lately. I'm not sure why this would conflict given the tests were both async false, but giving it a different dir name resolved some test issues. Thanks for the PR!